pennylane/transforms/intermediate_reps/rowcol.py

```diff
@@ -0,0 +1,446 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""CNOT routing algorithm RowCol as described in https://arxiv.org/abs/1910.14478."""
+from __future__ import annotations
```

Comment:
I believe this is no longer required now that we have 3.11 as the minimum required python version.

Reply:
its required otherwise I get an error from the `TYPE_CHECKING` bit

Reply:
I'm on python 3.13 locally